### DISCUSSION TITLE
AddQuad Method not working

### DISCUSSION THREAD

**Discussion by RadFad1** [leap71/PicoGK#57] [Coding for (Computational) Engineers]
The "msh.AddQuad" method has not been included in the Mesh class in the latest release of PicoGk.
![Screenshot (458)](https://github.com/user-attachments/assets/211c134c-a6a2-4edf-b554-6a85b32e499e)

**Comment by LinKayser** [comment]
Dear @RadFad1 

you have to clone the current version of PicoGK, as the function is not yet part of the release (I'll do it in the next couple of days).

If you clone the example code from: https://github.com/LinKayser/Coding4Engineers it should work.

Best,
Lin
